[documentation]: Review Genesis Block and Keys topics (#57)
hyperledger/iroha-2-docs: src/guide/configure/keys.md

# Public Key Cryptography

> It is a good practice to worry about public key cryptography, so we'll give

Reviewer: ```suggestion
It's OK to feel worried about public key cryptography. This is a complex topic that requires a lot of care and attention. To work with Iroha you _do_ need to have a basic understanding of how it works, so we provided you with a brief primer. 
```

> you a primer.

Reviewer: ```suggestion

```

# Public Key Cryptography

> Now it's a good practice to worry about public key cryptography, so we'll

Reviewer 2: ```suggestion
It is a good practice to worry about public key cryptography, so we'll
```

first block gets committed.

:::

### 3. Register a non-genesis account

Finally, while the first client _could_ use the genesis account to register
new users, it's not a great idea for private networks. You should, instead,
register a non-genesis account (for example _alice_@wonderland) and
`unregister` the genesis account.

::: warning NB

`iroha_client_cli` currently doesn't support unregister instructions. If
you plan on creating a private blockchain, you should consider writing your
own client based on the `client` Rust crate, or any of the provided client
libraries: [iroha-python](https://github.com/hyperledger/iroha-python),
[iroha-iOS](https://github.com/hyperledger/iroha-ios),
[iroha-java](https://github.com/hyperledger/iroha-java), or
[iroha-javascript](https://github.com/hyperledger/iroha-javascript/tree/iroha2).

:::

## Keys on the Client Side

Finally, let's talk about how keys are used in the client.

Every transaction is signed on behalf of some user, thus every operation
requires a key. That doesn't mean that you need to explicitly provide a key
every time.

For example, you need to have a user to register a user (just like you need

> scissors to open a bag with new scissors). But in order to register a user,

Reviewer 2: ```suggestion
scissors to cut off the tag from a pair of new scissors). But in order to register a user,
```
